websockets/extended-payload-length.any.js

```diff
@@ -1,13 +1,5 @@
-<!doctype html>
-<title>WebSockets : Boundary-value tests for the 'Extended payload length' field in RFC6455 section5.2 'Base Framing Protocol'</title>
-<meta name=timeout content=long>
-<script src=/resources/testharness.js></script>
-<script src=/resources/testharnessreport.js></script>
-<script src=constants.js?pipe=sub></script>
-<meta name="variant" content="">
-<meta name="variant" content="?wss">
```

Comment:
Yes. They cause the runner to run the test once with nothing appended to the URL and once with `?wss` appended (which tests `wss:` instead of `ws:`).